style.css

```diff
@@ -1,324 +1,630 @@
-  line-height: 2.4;
+    font-family: "Roboto Slab", serif;
+    font-size: 20px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 2.4;
 }
 .advert__subtitle {
-  margin-bottom: 43px;
+    margin-bottom: 43px;
 
-  text-align: center;
-  letter-spacing: -1.25px;
+    text-align: center;
+    letter-spacing: -1.25px;
 
-  color: #0d3351;
+    color: #0d3351;
 
-  font-family: "Roboto Slab", serif;
-  font-size: 50px;
-  font-weight: normal;
-  font-style: normal;
-  font-stretch: normal;
-  line-height: 0.96;
+    font-family: "Roboto Slab", serif;
+    font-size: 50px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: 0.96;
 }
 .advert__choice-button {
-  width: 202px;
-  margin-bottom: 5px;
-  padding: 16px 49px;
-
-  text-align: center;
-
-  color: #ffffff;
-  border: none;
-  background-color: #33c7fd;
-
-  font-family: "Raleway", sans-serif;
-  font-size: 14px;
-  font-weight: normal;
-  font-style: normal;
-  font-stretch: normal;
+    width: 202px;
+    margin-bottom: 5px;
+    padding: 16px 49px;
+
+    cursor: pointer;
+    text-align: center;
+
+    color: #ffffff;
+    border: none;
+    background-color: #33c7fd;
+
+    font-family: "Raleway", sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    font-style: normal;
+    font-stretch: normal;
+}
+.advert__choice-button:hover,
+.advert__choice-button:active,
+.advert__choice-button:focus {
+    background-color: darkblue;
 }
 .catalog {
-  display: flex;
-  flex-direction: column;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+        flex-direction: column;
 
-  width: 100%;
-  padding: 15px 0;
+    width: 100%;
+    padding: 15px 0;
 
-  background: #f3f3f3;
+    background: #f3f3f3;
 
-  align-items: center;
+    align-items: center;
+    -webkit-box-align: center;
+    -webkit-box-direction: normal;
+    -webkit-box-orient: vertical;
+    -ms-flex-align: center;
+    -ms-flex-direction: column;
 }
 .catalog__title {
-  text-align: center;
+    text-align: center;
 
-  color: #0d3351;
+    color: #0d3351;
 
-  font-family: "Roboto Slab", serif;
-  font-size: 20px;
+    font-family: "Roboto Slab", serif;
+    font-size: 20px;
 }
 .catalog__sort {
-  text-align: center;
+    text-align: center;
 
-  color: #0d3351;
+    color: #0d3351;
 
-  font-family: "Raleway", sans-serif;
-  font-size: 14px;
+    font-family: "Raleway", sans-serif;
+    font-size: 14px;
 }
 .catalog__sort_list {
-  display: inline-block;
+    display: inline-block;
 
-  width: auto;
-  max-width: 70px;
+    width: auto;
+    max-width: 70px;
 
-  border: none;
-  background: transparent;
+    cursor: pointer;
 
-  font-style: italic;
+    border: none;
+    background: transparent;
+
+    font-style: italic;
+}
+.catalog-list {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+        flex-direction: column;
+
+    -webkit-box-direction: normal;
```

Comment:
why do you use -webkit properties everywhere?

Reply:
Bacause of autoprefixer)
i'll fix this